src/commands/pull.js

```diff
@@ -0,0 +1,69 @@
+const chalk = require('chalk');
+const mkdirp = require('mkdirp');
+const fs = require('fs');
+const path = require('path');
+const tiged = require('tiged');
+const { ApifyCommand } = require('../lib/apify_command');
+const { getLoggedClientOrThrow } = require('../lib/utils');
+
+class PullCommand extends ApifyCommand {
+    async run() {
+        const { args } = this.parse(PullCommand);
+        const apifyClient = await getLoggedClientOrThrow();
+        const cwd = process.cwd();
+
+        const { actorId } = args;
+        const actor = await apifyClient.actor(actorId).get();
+        if (!actor) throw new Error(`Cannot find actor with ID '${actorId}' in your account.`);
+        const { name, versions } = actor;
+
+        // TODO: Check is sorted chronologically
+        const lastVersion = versions[versions.length - 1];
+
+        const dirpath = path.join(cwd, name);
+        mkdirp.sync(dirpath);
+
+        switch (lastVersion.sourceType) {
+            case 'TARBALL':
+            case 'SOURCE_FILES': {
+                const { sourceFiles } = lastVersion;
+                for (const file of sourceFiles) {
+                    if (file.folder) mkdirp.sync(path.join(cwd, name, file.name));
+                    else fs.writeFileSync(`${dirpath}/${file.name}`, file.content);
```

Comment:
This will fail if the file is in a nested folder (like `.actor/actor.json`) but its parent directory is not specified in the source files (or it's specified only after the file), because then the `writeFileSync` will fail because the directory into which it will try to write will not be created (yet).